working/augmentation-libraries/feature-specification.md

```diff
@@ -239,12 +242,13 @@ augment void trace() {
 ```
 
 The merge order is `main.dart`, `a.dart`, `b.dart`, then `c.dart`. The
-declarations in those libraries&mdash;new declarations or augmentations&mdash;
-are processed in that order, and source order within that.
+declarations in those library augmentations
+&mdash;new declarations or augmentations&mdash; are processed in that order,
+and source order within that.
 
 This order is user-visible in two ways:
 
-*   A non-augmenting declaration must appear first before it can be augmented.
+*   A regular declaration must appear first before it can be augmented.
```

Comment:
Nit: I do like "non-augmenting" here because it's a little less vague than "regular", which isn't explicitly defined.

Or maybe do:

> A regular (i.e. not augmenting)...